rocAL/include/pipeline/master_graph.h

```diff
@@ -102,6 +102,9 @@ class MasterGraph {
     Status run();
     Timing timing();
     RocalMemType mem_type();
+    RocalBatchPolicy last_batch_policy();
+    bool last_batch_padded();
```

Comment:
please combine 105 and 106 into a single api call. For e.g if last_batch_padded_size is 0, last_batch_padded is false

Reply:
Combined last_batch_policy() and last_batch_padded() function to return a std::pair<RocalBatchPolicy, bool>